Packages/com.unity.inputsystem/InputSystem/Plugins/XR/TrackedPoseDriver.cs

```diff
@@ -342,8 +446,17 @@ bool HasStereoCamera()
         }
 
         protected virtual void PerformUpdate()
-        {
-            SetLocalTransform(m_CurrentPosition, m_CurrentRotation);
+        {            
+            PoseDataFlags trackingStateFlags = m_CurrentTrackingStateFlags;
+            if (m_TrackingStateFlags != null && m_TrackingStateFlags.action != null && m_TrackingStateFlags.action.bindings.Count > 0)
+            { 
+            }   
+            else
+            {
+                trackingStateFlags = PoseDataFlags.Position | PoseDataFlags.Rotation;
+            }
```

Comment:
I feel like the following is a bit clearer and simpler for readability: 
```
if (m_TrackingStateFlags == null || m_TrackingStateFlags.action == null || m_TrackingStateFlags.action.bindings.Count == 0)
    trackingStateFlags = PoseDataFlags.Position | PoseDataFlags.Rotation;
```

Is there a performance advantage to the existing code that is relevant?